skl2onnx/operator_converters/_gp_kernels.py

```diff
@@ -0,0 +1,255 @@
+# -------------------------------------------------------------------------
```

Comment:
Please add unit tests for this file. Current coverage is 12%.
skl2onnx/operator_converters/_gp_kernels.py                  121    107    12%   23-24, 28-61, 66-73, 78-88, 97-104, 113-121, 126-232, 238-255

Reply:
Unit tests do exists. They are just disable since they cannot be run with the current runtime.